rewrite feature guide (#16)
qwat/docs: features-guide/index.rst

QWAT / TEKSI Drinking Water
===========================

> QWAT is a PostgreSQL - PostGIS data model almost compatible with the swiss norm SIA405. The data are accessed and edited with a dedicated QGIS project.

Reviewer: ```suggestion
The TEKSI drinking water module (project name: QWAT) is a decision support tool that allows you to:

* manage and map the data of your drinking water network with all its components such as pipes, valves, pumps, tanks, etc.
* indicate the characteristics of network objects in the form of attributes such as diameter, material, installation depth, leaks, date of installation, etc.
* produce plans and extract statistics from the database such as network value, total length of pipes, identification of future interventions, etc.
* export geodata in compliance with the requirements of Swiss standards

QWAT is a complete open source module based on a PostgreSQL - PostGIS data model almost compatible with the swiss norm SIA405. The data are accessed and edited with a dedicated QGIS project.
```

@tproduit  it would be a great thing to include website introduction to the module!

Author: What do you mean with "website introduction"?

Reviewer: A similar introduction which is provided on [www.teksi.ch](https://www.teksi.ch/modules/eau-potable/) but extended here.

> The project is maintained by the `community TEKSI <https://github.com/qwat/extension_sire>`_.

Reviewer: ```suggestion
The project is maintained by the `TEKSI community <https://www.teksi.ch>`_.
```

Main features
-------------

> Data model

Reviewer: ```suggestion
Network objects
```

^^^^^^^^^^
The data model contains all the objects of a drinking water network and their attributes:

* Pipe
* Tank
* Valve
* Chamber
* Cover
* Treatment
* Pump
* Pressure control
* Source
* Meter
* Hydrant
* Subscriber
* Sampling point
* Leaks

> Their geometries contain the Z value (altitude).

Reviewer: ```suggestion
Their geometries contain the Z value (altitude) which can be edited via the native QGIS tools.
```

> Their geometries contain the Z value (altitude) which can be edited via the native QGIS tools.

Reviewer: ```suggestion
The altitude of these objects is handled directly in their geometries which can be modified by the native QGIS tools.
```

The data model is almost compatible with the SIA405.

> The model is comprehensive and provide all the common objects and atttributes.

Reviewer: ```suggestion
The model is comprehensive and provide all the common objects and attributes.
```
typo

> However, there is the possibility to add your own attributes or objects.

Reviewer: ```suggestion
However, there is the possibility to customize attributes or objects and suggest modifications to the community.
```

Viewing and editing
^^^^^^^^^^^^^^^^^^^^
A preconfigured QGIS project is provided. The network objects are shown with a standard symbology. 

> The editing can be done directly from QGIS, each object has a corresponding form that eases the edition of the attributes.

Reviewer: ```suggestion
The editing can be done directly from QGIS, each object has a corresponding form that eases the edition of attributes.
```
